sdk/webpubsub/Azure.Messaging.WebPubSub/src/WebPubSubServiceClientOptions.ServiceVersion.cs

```diff
@@ -24,10 +24,12 @@ public partial class WebPubSubServiceClientOptions : ClientOptions
         public enum ServiceVersion
         {
 #pragma warning disable CA1707 // Identifiers should not contain underscores
+#pragma warning disable AZC0016 // Invalid ServiceVersion member name.
             /// <summary> The 2021_05_01_preview version of the Azure WebPubSub service. </summary>
             V2021_05_01_preview = 1,
             /// <summary> The 2021_08_01_preview version of the Azure WebPubSub service. </summary>
             V2021_08_01_preview = 2,
+#pragma warning restore AZC0016 // Invalid ServiceVersion member name.
 #pragma warning restore CA1707 // Identifiers should not contain underscores
```

Comment:
For this enum which is non-generated, could we change this to something like -

```
    public enum ServiceVersion
    {
        /// <summary> The 2021_05_01_preview version of the Azure WebPubSub service. </summary>
        [System.ComponentModel.EditorBrowsable(System.ComponentModel.EditorBrowsableState.Never)]
        V2021_05_01_preview = 1,
#pragma warning disable CA1069 // Enums values should not be duplicated
        V2021_05_01_Preview = 1,
#pragma warning restore CA1069 // Enums values should not be duplicated
        /// <summary> The 2021_08_01_preview version of the Azure WebPubSub service. </summary>
        [System.ComponentModel.EditorBrowsable(System.ComponentModel.EditorBrowsableState.Never)]
        V2021_08_01_preview = 2,
#pragma warning disable CA1069 // Enums values should not be duplicated
        V2021_08_01_Preview = 2,
#pragma warning restore CA1069 // Enums values should not be duplicated
    }
```

Reply:
I've opened bugs against owners for considerations like that. If they haven't shipped yet, they can simply rename it. It's a good suggestion, though. I'll add a reference to this comment to each of them.

Reply:
![image](https://user-images.githubusercontent.com/1532486/134260299-59dcf612-739a-4295-9541-431cbf605f3d.png)